Basic/2DSpaceShooter/Assets/Scripts/ShipControl.cs

```diff
@@ -107,7 +107,7 @@ public class ShipControl : NetworkBehaviour
     float m_OldSpin = 0;
 
     // server movement
-    private NetworkVariable<float> m_Thrusting = new NetworkVariable<float>();
+    float m_Thrusting;
```

Comment:
While we're here, might be better to rename to `m_Thrust` or `m_Speed`

Reply:
Done!
I had to rename a particle system that used the `m_Thrust` name, but this is clearer yeah.